apps/wallet-mobile/src/features/Swap/SwapNavigator.tsx

```diff
@@ -62,9 +64,37 @@ export const SwapTabNavigator = () => {
           })}
           style={styles.tab}
         >
-          <Tab.Screen name="token-swap" component={StartSwapOrderScreen} />
+          <Tab.Screen
+            listeners={{
```

Comment:
Seems like an iOS only bug, could it be happening in other screens? The maybe abstract the whole listener so it can be spread into any Tab.Screen?